### PR TITLE
Fix jenkins autodeploy on int

### PR DESCRIPTION
**Pull request by oterral**
Fix #100

<jenkins>[Test link](https://mf-geoadmin3.int.bgdi.ch/teo_make/index.html)</jenkins>

### CODE REVIEW

File: scripts/clonebuild.sh [hunk 1]
```diff
@@ -29,39 +24,25 @@ DEPLOY_GIT_BRANCH=${@:$OPTIND+2:1}
 DEEP_CLEAN=${@:$OPTIND+3:1}
 NAMED_BRANCH=${@:$OPTIND+4:1}
 
+# Remove the clone folder
+if [ $DEEP_CLEAN = true ]; then
+  make cleanall
+fi
```
Comment (loicgasser) [inline scripts/clonebuild.sh]: we want to `cleanall` in the snapshot not locally.

File: scripts/clonebuild.sh [hunk 2]
```diff
@@ -29,39 +24,25 @@ DEPLOY_GIT_BRANCH=${@:$OPTIND+2:1}
...
-  export KEEP_VERSION="false"
-  make .build-artefacts/last-version
-  S3_BASE_PATH=$(get_s3_basepath $DEPLOY_GIT_BRANCH)
-fi
-S3_SRC_BASE_PATH=$S3_BASE_PATH"src/"
-
-echo "S3_BASE_PATH:"
-echo $S3_BASE_PATH
-echo "S3_SRC_BASE_PATH:"
-echo $S3_SRC_BASE_PATH
```
Comment (loicgasser) [inline scripts/clonebuild.sh]: I think you could still display these. It's useful to know what's going on.

File: Makefile [hunk 3]
```diff
@@ -397,7 +409,7 @@ flushvarnish: guard-DEPLOY_TARGET
 
 # This internal target has been created to have the good global variable values
 # from rc_XXX file.
-flushvarnishinternal: guard-VARNISH_HOSTS guard-API_URL guard-E2E_TARGETURL
+flushvarnishinternal: guard-API_URL guard-E2E_TARGETURL
```
Comment (loicgasser) [inline Makefile]: why remove `guard-VARNISH_HOSTS`?

File: scripts/clonebuild.sh (hunk 1 above)
Comment (oterral) [reply scripts/clonebuild.sh]: `cleanall` remove the entire snapshot

File: scripts/clonebuild.sh (hunk 2 above)
Comment (oterral) [reply scripts/clonebuild.sh]: They are set in the makefile. I'm not sure they are accessible outside

File: Makefile (hunk 3 above)
Comment (oterral) [reply Makefile]: rc_ dev hasn't VARNISH_HOSTS

Comment (loicgasser) [reply Makefile]: You can use an empty list for dev

File: scripts/clonebuild.sh (hunk 2 above)
Comment (loicgasser) [reply scripts/clonebuild.sh]: Ok

File: scripts/clonebuild.sh (hunk 1 above)
Comment (loicgasser) [reply scripts/clonebuild.sh]: Ok I hadn t see that. But still you are in the current directory and your also removing your built artefacts. Just remove the clonedir instead.

File: scripts/clonebuild.sh (hunk 2 above)
Comment (oterral) [reply scripts/clonebuild.sh]: We can display it when we execute a  Make target.

File: scripts/clonebuild.sh (hunk 1 above)
Comment (oterral) [reply scripts/clonebuild.sh]: yes good idea

File: Makefile (hunk 3 above)
Comment (oterral) [reply Makefile]: ok

File: scripts/clonebuild.sh (hunk 1 above)
Comment (oterral) [reply scripts/clonebuild.sh]: done

File: scripts/clonebuild.sh (hunk 2 above)
Comment (oterral) [reply scripts/clonebuild.sh]: I've added a showVariable target in make . It is used on each beginning of build or deploy targets

File: Makefile (hunk 3 above)
Comment (oterral) [reply Makefile]: I have still keep the guard-VARNISH_HOSTS but I have mprove the log message when VARNISH_HOST is empty

Comment (oterral) [reply Makefile]: because "export VARNISH_HOSTS=" in rc_dev was detected as a failure

Comment (loicgasser) [reply Makefile]: I was thinking `export VARNISH_HOSTS=()` but you changed that and it would be `export VARNISH_HOSTS=""`

LGTM

Comment (oterral) [reply Makefile]: it didn't work with VARNISH_HOSTS="", I've alaso tried with a () but didn't work too. You can try if you want.

**Comment by loicgasser** [comment]
wrong order of merge I'll fix

**Pull request merged**